feature_engine/selection/drop_features.py

```diff
@@ -81,10 +81,11 @@ def fit(self, X: pd.DataFrame, y: pd.Series = None):
         # present in the df.
         X[self.features_to_drop]
 
-        self.features_to_drop_ = self.features_to_drop
+        self.features_to_drop_ = _find_all_variables(X, variables=self.features_to_drop)
+        # self.features_to_drop_ = self.features_to_drop
```

Comment:
could you please remove the commented line?

Reply:
Hi,

I´ve pushed a new commit with the formatting changes you asked for, and an expanded version of the test test_drop_1_variable(...) which demonstrates the fix ( len('Marks') = len(X.columns) which was triggering the bug).